openerp/addons/base/module/module.py

```diff
@@ -391,7 +392,7 @@ def state_update(self, newstate, states_to_update, level=100):
             # check dependencies and update module itself
             self.check_external_dependencies(module.name, newstate)
             if module.state in states_to_update:
-                module.write({'state': newstate, 'demo': module_demo})
+                module.write({'state': newstate, 'demo': module_demo, 'installation_date': fields.datetime.now()})
```

Comment:
why installed_date set in state_update method ? I think, it should be set in button_install method.